apps/s_client.c

```diff
@@ -2172,6 +2172,15 @@ int s_client_main(int argc, char **argv)
              * HTTP/d.d ddd Reason text\r\n
              */
             mbuf_len = BIO_gets(fbio, mbuf, BUFSIZZ);
+            if (mbuf_len < sizeof("HTTP/1.0 200") - 1) {
```

Comment:
This is exactly the reason why I'd rather see strlen. It's hard to remember if sizeof("a") accounts for trailing zero or not. Code suggests that it does, but what does this print
```
int main()
{
    printf("%d\n", strlen("a"));
}
```